Scripts/UI/GradientButton.cs

```diff
@@ -6,18 +6,22 @@
 using UnityEditor.Experimental.EditorVR.Utilities;
 using UnityEngine;
 using UnityEngine.EventSystems;
+using UnityEngine.Serialization;
 using UnityEngine.UI;
 
 namespace UnityEditor.Experimental.EditorVR.UI
 {
 	sealed class GradientButton : MonoBehaviour, IPointerEnterHandler, IPointerExitHandler, IPointerClickHandler
 	{
-		const float k_IconHighlightedLocalZOffset = -0.0015f;
 		const string k_MaterialAlphaProperty = "_Alpha";
 		const string k_MaterialColorTopProperty = "_ColorTop";
 		const string k_MaterialColorBottomProperty = "_ColorBottom";
 
 		public event Action click;
+		public event Action hoverEnter;
+		public event Action hoverExit;
+		public event Action highlightStart;
```

Comment:
highlightStart/End are not used outside of GradientButton. I vote for removing them. What were/are they for?